content/docs/command-reference/dag.md

```diff
@@ -1,49 +1,49 @@
 # dag
 
-Show [stages](/doc/command-reference/run) in a pipeline that lead to the
-specified stage. By default it lists
-[DVC-files](/doc/user-guide/dvc-files-and-directories).
+Display connected [stages](/doc/command-reference/run) in the pipelines that
+contains them.
```

Comment:
```suggestion
contain them.
```
?

Reply:
Also, is `that contain them` really necessary? Stages belong to pipelines by definition.

Reply:
Yes but there could be more than one pipeline right? This command shows the one that contains the target stage (if a target is provided).

Reply:
But yeah this could be simplifies as much as just "Displays pipeline(s)."

The help output reads `Visualize DVC project DAG.` though. We should try to have these descriptions match. Isn't "pipeline" still the preferred term?

Reply:
Yes, there could be more than one pipeline.

DAG and pipeline(s) are not really conflicting, not sure there should be a prefered term necessarily.

Reply:
From multiple experiences with key terminology, it's best to avoid synonyms. They can cause confusion and make docs harder to maintain. Unfortunately I don't see us moving away from "pipeline" and the command is now called `dag` so these will have to coexist but still I would prefer "pipelines" as much as possible (and some links to "DAG" concept [Wikipedia] and `dvc dag` of course).